app/models/manageiq/providers/openstack/network_manager.rb

```diff
@@ -36,6 +36,7 @@ class ManageIQ::Providers::Openstack::NetworkManager < ManageIQ::Providers::Netw
            :hostname,
            :default_endpoint,
            :endpoints,
+           :supports_events?,
```

Comment:
Since event support is tied to the endpoint, any manager which delegates `verify_credentials` and `connect` should also delegate if events are supported.